nav2_costmap_2d/src/costmap_2d_markers.cpp

```diff
@@ -111,7 +112,7 @@ void voxelCallback(const nav2_msgs::msg::VoxelGrid::ConstSharedPtr grid)
   visualization_msgs::msg::Marker m;
   m.header.frame_id = frame_id;
   m.header.stamp = stamp;
-  m.ns = g_marker_ns;
+  m.ns = g_node->get_namespace();
```

Comment:
This one is not just a formatting change. Just wanted to draw your attention to it to make sure it was what you intended.

Reply:
I received a lint error that there couldn't be a global string variable, so this was my attempt to replace it.